modules/nw-about-configuring-master-interface-container.adoc

```diff
@@ -0,0 +1,175 @@
+ priority: 99
+ resourceName: sriovnic
+ nodeSelector:
+    feature.node.kubernetes.io/network-sriov.capable: "true"
+----
+
+.. Apply the YAML by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f sriov-node-network-policy.yaml
+----
+
+. Create an SR-IOV network:
+
+.. Create the `SriovNetwork` custom resource (CR) for the additional SR-IOV network attachment as in the following example CR. Save the YAML as the file `sriov-network-attachment.yaml`:
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+ name: sriov-network
+ namespace: openshift-sriov-network-operator
+spec:
+ networkNamespace: test-namespace
+ resourceName: sriovnic
+ spoofChk: "off"
+ trust: "on"
+----
+
+.. Apply the YAML by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f sriov-network-attachment.yaml
+----
+
+. Create a YAML file for the VLAN additional network configuration and then save the YAML in the `vlan100-additional-network-configuration.yaml` file:
++
+[source,yaml]
+----
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  name: vlan-100
+  namespace: test-namespace
+spec:
+  config: |
+    {
+      "cniVersion": "0.4.0",
+      "name": "vlan-100",
+      "plugins": [
+        {
+          "type": "vlan",
+          "master": "ext0", <1>
+          "mtu": 1500,
+          "vlanId": 100,
+          "linkInContainer": true, <2>
+          "ipam": {"type": "whereabouts", "ipRanges": [{"range": "1.1.1.0/24"}]}
+        }
+      ]
+    }
+----
++
+<1> The VLAN configuration needs to specify the master name. This can be configured in the pod networks annotation.
+<2> The `linkInContainer` parameter must be specified.
+
+. Apply the YAML by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f vlan100-additional-network-configuration.yaml
+----
+
+. Create a pod definition by using the earlier specified networks and then save the YAML in the `pod-a.yaml` file. The pod definition must use the following networks annotation:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx-pod
+  annotations:
+    k8s.v1.cni.cncf.io/networks: '[
+      {
+        "name": "sriov-network",
+        "namespace": "test-namespace",
+        "interface": "ext0" <1>
+      },
+      {
+        "name": "vlan-100",
+        "namespace": "test-namespace",
+        "interface": "ext0.100"
+      }
+    ]'
+spec:
+  securityContext:
+    runAsNonRoot: true
+  containers:
+    - name: nginx-container
+      image: nginxinc/nginx-unprivileged:latest
+      securityContext:
+        allowPrivilegeEscalation: false
+        capabilities:
+          drop: ["ALL"]
+      ports:
+        - containerPort: 80
+      seccompProfile:
+        type: "RuntimeDefault"
+----
++
+<1> The name to be used as the master for the VLAN interface.
+
+. Apply the YAML by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f pod-a.yaml
```

Comment:
When I applied this manifest I've got following error:
`Error from server (Forbidden): error when creating pod-a.yaml": pods "nginx-pod" is forbidden: violates PodSecurity "restricted:v1.24": runAsNonRoot != true (pod must not set securityContext.runAsNonRoot=false), seccompProfile (pod or container "nginx-container" must set securityContext.seccompProfile.type to "RuntimeDefault" or "Localhost")`

I had to add following labels in order to make it work:
```
oc label namespace test-namespace pod-security.kubernetes.io/enforce=privileged --overwrite
oc label namespace test-namespace pod-security.kubernetes.io/audit=privileged
oc label namespace test-namespace pod-security.kubernetes.io/warn=privileged --overwrite
oc label namespace test-namespace security.openshift.io/scc.podSecurityLabelSync=false --overwrite 
```